### ISSUE TITLE
Testing with the best model

### ISSUE THREAD

**Issue by LeeDoYup** [google-research/augmix#8]
I found that cifar.py code does not test the performance with the best model.
https://github.com/google-research/augmix/blob/7c84885fe064435b4b7a8b596b937f0a879e458c/cifar.py#L431

In addition, why the validation set of CIFAR is not used in the model selection ??

**Comment by cygerts** [comment]
I think they choose the model from 100th epoch.
From the paper: "The All Convolutional Network and Wide ResNet train for 100 epochs,
and the DenseNet and ResNeXt require 200 epochs for convergence"

**Comment by LeeDoYup** [comment]
I think it's much unfair, because the code evenly save the best model.
https://github.com/google-research/augmix/blob/7c84885fe064435b4b7a8b596b937f0a879e458c/cifar.py#L413-L414

We can just use the best model in evaluation :) !!

**Comment by LeeDoYup** [comment]
I also checked, the selection of the best model doesn't change augmix's performances on test_c.

**Comment by normster** [comment]
Hi @LeeDoYup, there isn't much of a difference between the best model and last model so we'll keep the best model for

**Comment by normster** [comment]
Hi @LeeDoYup, there isn't much of a difference between the best and latest models so we just evaluate on the latest model checkpoint.
